py/htm/advanced/regions/__init__.py

```diff
@@ -0,0 +1,36 @@
+
+from collections.abc import Iterable
+
+def executeCommand(command_name, region_instance, *args):
```

Comment:
is this the "enablement" you've discussed with @dkeeney ? This code looks quite generic, would it be possible to add it to the "main" engine (or relevant place)?

Reply:
Yes. It is generic but its home should be in Python because it provides a generic entry for expanding the API of PyRegion implementations.

Reply:
> would it be possible to add it to the "main" engine 

This is in the main engine in the sense that executeCommand( ) is part of the API and is available for use by C++ region implementations.  What he did was expose it to python.  That was an over-site on my part.  It should have been there all along.  Thank you @fcr.

Reply:
> executeCommand( ) is part of the API and is available for use by C++ region implementations. What he did was expose it to python

yes, yet here it's still under `htm.advanced.regions` what I'm trying to figure is, if we can add it to `PyRegionImpl` (in the bindings for a generic Region class, or some other suitable place)?

Reply:
@breznak  Oh, I total missed the point you were making.  I had to go look at this again.

Yes, I agree, we should make those changes available to all python users, not just those using the advanced section.  The C++ signature for the call is:
```
std::string executeCommand(const  std::string& command, vector<std::string>)
```
If I am following this correctly, @fcr is making a python function that accepts a normal argument list and creates a vector of strings so it will match the C++ signature.

What we probably should do is implement that in the bindings. `bindings/py/cpp_src/bindings/engine/py_Engine.cpp`   in the `Region` section where it exposes the executeCommand.   We probably want to keep the existing calling signature and add a new overload that accepts *args rather than the vector.

@fcr, I can help with that if you do not know how to do it.

Reply:
My C++ skills are fairly weak so yes, it would be better if you were to do this.

Reply:
My Python skills are fairly weak to together we could make a good team :-)
Ok, let me make a new PR.  Then when that is merged we can try your PR to make sure this works with your PR and then your PR would be ready to merge (if all else is ok).

Reply:
Tag me David, and I'll merge the PR asap